### DISCUSSION TITLE
"containerd-rootless.sh needs to be present under $PATH"

### DISCUSSION THREAD

**Discussion by jack-liew** [containerd/nerdctl#2231] [Q&A]
OS: ubuntu 23.04
Kubernetes: k3s

I installed k3s, use ctr can pull images normally.

when i download nerdctl and just exec nerdctl,  It prompts:
`rootless containerd not running? (hint: use `containerd-rootless-setuptool.sh install` to start rootless containerd): stat /run/user/1000/containerd-rootless: no such file or directory`

then i exec `./containerd-rootless-setuptool.sh install`, it prompts:
`containerd-rootless.sh needs to be present under $PATH`

I don not need rootless, and can not understand why.
---
Thanks

**Comment by yuchanns** [comment]
If you do not need rootless, just execute your nerdctl command with `sudo`
